.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Set up node
         uses: actions/setup-node@v2
         with:
-          node-version: lts/*
+          node-version: 16.17.1
```

Comment:
Can you pin only the major version?

Reply:
odd versions are never LTS, and 18 doesn't work :/
you want 16.0.0?

Reply:
Can you put just `16.x` or something like that?